contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -194,6 +165,30 @@ contract ConfiguredDualAggregatorBaseTest is DualAggregatorBaseTest {
   }
 }
 
+contract RoundDataDualAggregatorBaseTest is ConfiguredDualAggregatorBaseTest {
+  int192[] internal answers = [int192(10), int192(11), int192(12), int192(13), int192(14), int192(15)];
+  uint32[] internal observationsTimestamps = [uint32(1), uint32(6), uint32(11), uint32(16), uint32(21), uint32(26)];
+  uint32[] internal recordedTimestamps = [uint32(5), uint32(10), uint32(15), uint32(20), uint32(25), uint32(30)];
```

Comment:
These state variables should be named according to the solidity style guide, s_answers, s_observationsTImestamps, s_recordedTimestamps

Reply:
Do they need to be state variables?